packages/filter.punches/src/index.ts

```diff
@@ -5,64 +5,74 @@ import {
 
 var sproto = String.prototype
 
-var filters = {}
-
-// Convert value to uppercase
-filters.uppercase = function (value) {
-  return sproto.toUpperCase.call(unwrap(value))
+interface Filters {
```

Comment:
many consumers of Knockout/TKO will extend the filters, so this type has to be exposed for others to extend it.

Reply:
export is added, see https://github.com/Auge19/tko/pull/4